testing_and_setup/compass/ocean/jigsaw_to_MPAS/build_mesh.py

```diff
@@ -37,15 +36,15 @@ def build_mesh(preserve_floodplain=False, floodplain_elevation=20.0,
     print('Step 1. Build cellWidth array as function of latitude and '
           'longitude')
     cellWidth, lon, lat = define_base_mesh.cellWidthVsLatLon()
-    sio.savemat(
-        'cellWidthVsLatLon.mat', {
-            'cellWidth': cellWidth, 'lon': lon, 'lat': lat})
+    da = xarray.DataArray(cellWidth,
+                          dims=['lat','lon'],
+                          coords={'lat':lat,'lon':lon},
+                          name='cellWidth')
+    da.to_netcdf('cellWidthVsLatLon.nc')
+    
 
-    print('Step 2. Build mesh using JIGSAW')
-    args = ["octave", "--silent", "--eval",
-            "jigsaw_driver"]
-    print("running", ' '.join(args))
-    subprocess.check_call(args, env=os.environ.copy())
+    print('Step 2. Generate mesh with JIGSAW')
+    jigsaw_driver(cellWidth, lon, lat)
```

Comment:
Where is this function specified?

Reply:
The reason I ask is because I don't see an import statement at the top of this function.